test/ui/test.js

```diff
@@ -57,11 +58,13 @@ test.describe('UI tests', function() {
 
   // build the selenium webdriver
   before(function() {
+    this.timeout(initTimeout);
     driver = new selenium.Builder()
       .forBrowser('chrome')
       .usingServer('http://localhost:4444/wd/hub')
       .build();
 
+    driver.manage().timeouts().setScriptTimeout(5000);
```

Comment:
Please add a variable for this constant, probably up with the other timeout variables.